### DISCUSSION TITLE
New block in Scratch Addons: "set z to ()"

### DISCUSSION THREAD

**Discussion by Indious111** [ScratchAddons/ScratchAddons#4626] [Ideas]
I see a lot of people wanted a "set z to" block, which means setting how the 3d looks, so then it will be much easier to make a 3d project, and I know this idea was rejected in Scratch 3.0, but this would be great to put in Scratch Addons!

**Comment by Endermaster3690** [comment]
how is this gonna work? the camera isn't in 3d

**Reply by Norbiros** [reply]
Something like that: `move sprite to z layer`. Something like `z-index` property in CSS

**Reply by Endermaster3690** [reply]
Well then how will it look to people without SA?

**Reply by Endermaster3690** [reply]
The blocks I mean

> Well then how will it look to people without SA?

**Reply by Chameleon-Coder** [reply]
I think maybe custom blocks like with debugger

**Reply by Endermaster3690** [reply]
i was asking, how will the script look?

**Reply by Chameleon-Coder** [reply]
Oh. My bad.

**Reply by DatGamerYoloYT** [reply]
z = layers

**Comment by rimopa** [comment]
I think a "go to layer ()" would be better, but still confusing and would need a system to non-sa users to work, using a system for set layers changing them with the default blocks, like a short
![scratchblocks](https://user-images.githubusercontent.com/97112832/171290415-5d1b5943-4780-429f-a747-d3fecda6cd71.png)

**Reply by Chameleon-Coder** [reply]
Yes

**Reply by Indious111** [reply]
I will be a looks block, making a costume be kinda like this: https://scratch.mit.edu/projects/22797110/

**Reply by Norbiros** [reply]
Why do you want to use this block?
You can just move to bottom, and then move z layers up.

**Reply by Endermaster3690** [reply]
> Why do you want to use this block? You can just move to bottom, and then move z layers up.

The workaround for the go to z is too easy.
